diff --git a/node_modules/@smui/checkbox/dist/Checkbox.svelte b/node_modules/@smui/checkbox/dist/Checkbox.svelte
index d60a338..3948abc 100644
--- a/node_modules/@smui/checkbox/dist/Checkbox.svelte
+++ b/node_modules/@smui/checkbox/dist/Checkbox.svelte
@@ -51,6 +51,9 @@
       : undefined}
     {...nativeControlAttrs}
     {...prefixFilter(restProps, 'input$')}
+    onclick={(e) => {
+      e.currentTarget.blur();
+    }}
     onblur={(e) => {
       dispatch(getElement(), 'blur', e);
       restProps.input$onblur?.(e);
