diff --git a/node_modules/@smui/icon-button/dist/IconButton.svelte b/node_modules/@smui/icon-button/dist/IconButton.svelte
index ca0c628..d8b2011 100644
--- a/node_modules/@smui/icon-button/dist/IconButton.svelte
+++ b/node_modules/@smui/icon-button/dist/IconButton.svelte
@@ -65,6 +65,7 @@
       dispatch(getElement(), 'SMUITopAppBarIconButtonNav');
     }
     restProps.onclick?.(e);
+    e.currentTarget.blur();
   }}
   ><div class="mdc-icon-button__ripple"></div>
   {@render children?.()}{#if touch}<div
