diff --git a/node_modules/@smui/icon-button/dist/IconButton.svelte b/node_modules/@smui/icon-button/dist/IconButton.svelte
index 75082d8..2f651c2 100644
--- a/node_modules/@smui/icon-button/dist/IconButton.svelte
+++ b/node_modules/@smui/icon-button/dist/IconButton.svelte
@@ -64,6 +64,7 @@
     if (context === 'top-app-bar:navigation') {
       dispatch(getElement(), 'SMUITopAppBarIconButtonNav');
     }
+    e.currentTarget.blur();
     restProps.onclick?.(e);
   }}
   ><div class="mdc-icon-button__ripple"></div>
