diff --git a/node_modules/@smui/textfield/dist/Input.svelte b/node_modules/@smui/textfield/dist/Input.svelte
index fa92f34..109ae70 100644
--- a/node_modules/@smui/textfield/dist/Input.svelte
+++ b/node_modules/@smui/textfield/dist/Input.svelte
@@ -140,6 +140,23 @@
     }
     switch (type) {
       case 'number':
+        value = toNumber(e.currentTarget.value);
+        if (e.currentTarget.value === '' || e.currentTarget.value === '0') {
+          if (e.data === '-') {
+            value = '-0';
+            e.target.value = '-0';
+          } else if (e.inputType === 'deleteContentBackward') {
+            value = '0';
+            e.target.value = '0';
+          } else if (e.inputType === 'insertFromPaste' && !isNaN(parseInt(e.data))) { 
+            value = parseInt(e.data);
+            e.target.value = parseInt(e.data);
+          } else {
+            value = '0';
+            e.target.value = '0';
+          }
+        }
+        break;
       case 'range':
         value = toNumber(e.currentTarget.value);
         break;
diff --git a/node_modules/@smui/textfield/dist/Textfield.svelte b/node_modules/@smui/textfield/dist/Textfield.svelte
index 5a2fba5..9f2a128 100644
--- a/node_modules/@smui/textfield/dist/Textfield.svelte
+++ b/node_modules/@smui/textfield/dist/Textfield.svelte
@@ -4,7 +4,7 @@
   <label
     bind:this={element}
     use:Ripple={{
-      ripple: !textarea && variant === 'filled',
+      ripple: variant === 'filled',
       unbounded: false,
       addClass,
       removeClass,
@@ -50,7 +50,7 @@
       'helperLine$',
     ])}
   >
-    {#if !textarea && variant !== 'outlined'}
+    {#if variant !== 'outlined'}
       {#if variant === 'filled'}
         <span class="mdc-text-field__ripple"></span>
       {/if}
@@ -68,7 +68,7 @@
         >
       {/if}
     {/if}
-    {#if textarea || variant === 'outlined'}
+    {#if variant === 'outlined'}
       <NotchedOutline
         bind:this={notchedOutline}
         noLabel={noLabel || label == null}
@@ -178,7 +178,7 @@
     <ContextFragment key="SMUI:textfield:icon:leading" value={false}>
       {@render trailingIcon?.()}
     </ContextFragment>
-    {#if !textarea && variant !== 'outlined' && ripple}
+    {#if variant !== 'outlined' && ripple}
       <LineRipple
         bind:this={lineRipple}
         {...prefixFilter(restProps, 'ripple$')}
