diff --git a/node_modules/@smui/button/dist/Button.svelte b/node_modules/@smui/button/dist/Button.svelte
index 9378004..28fc595 100644
--- a/node_modules/@smui/button/dist/Button.svelte
+++ b/node_modules/@smui/button/dist/Button.svelte
@@ -48,6 +48,7 @@
   {...restProps}
   onclick={(e: MouseEvent) => {
     handleClick();
+    e.currentTarget.blur();
     restProps.onclick?.(e);
   }}
   ><div class="mdc-button__ripple"></div>
