diff --git a/node_modules/@smui/switch/dist/Switch.svelte b/node_modules/@smui/switch/dist/Switch.svelte
index a816218..536cb25 100644
--- a/node_modules/@smui/switch/dist/Switch.svelte
+++ b/node_modules/@smui/switch/dist/Switch.svelte
@@ -33,6 +33,8 @@
       instance.handleClick();
     }
     restProps.onclick?.(e);
+    e.stopPropagation();
+    e.currentTarget.blur();
   }}
 >
   <div class="mdc-switch__track"></div>
